docs/upgrade/2.0.0_to_2.0.1.rst

```diff
@@ -42,21 +50,21 @@ update by running the following commands: ::
   cd ~/Persistent/securedrop
   git fetch --tags
   gpg --keyserver hkps://keys.openpgp.org --recv-key \
-   "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
-  git tag -v 1.8.1
+   "2359 E653 8C06 13E6 5295 5E6C 188E DD3B 7B22 E6A3"
+  git tag -v 2.0.1
 
 The output should include the following two lines: ::
 
-    gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+    gpg:                using RSA key 2359E6538C0613E652955E6C188EDD3B7B22E6A3
     gpg: Good signature from "SecureDrop Release Signing Key"
```

Comment:
Let's double-check the expected output here, I believe it may include the UID
> gpg: Good signature from "SecureDrop Release Signing Key <securedrop-release-key-2021@freedom.press>" [unknown]
It does when I verify other artifacts signed with the new key such as the signing key transition statement. I don't know if the `[unknown]` is expected in the output.